### PR TITLE
Drop personalization of device name suggestion

### PR DESCRIPTION
**Pull request by mlsteele**
iPad simulator was presenting as "iPad Pro (11-inch)" which is not a valid keybase device name. ~Drop the parens so it's "iPad Pro 11-inch" which is a valid device name.~

### CODE REVIEW

File: shared/constants/signup.tsx [hunk 1]
```diff
@@ -8,8 +8,11 @@ export const usernameHint =
 export const noEmail = 'NOEMAIL'
 export const waitingKey = 'signup:waiting'
 
-const defaultDevicename =
-  Platforms.realDeviceName ||
+// Remove parens from the device name. Keybase device names cannot have parens.
+const cleanupDeviceName = (name: string) => name.replace(/[()]/g, '')
```
Comment (chrisnojima) [inline shared/constants/signup.tsx]: this should be more expansive

Comment (mlsteele) [reply shared/constants/signup.tsx]: On the client here are the rules
- must match `^[a-zA-Z0-9][ _'a-zA-Z0-9+‘’—–-]*$`
- 3 <= len <= 64
- some stuff with curly quotes
- must not match `  |[ '_-]$|['_-][ ]?['_-]` (github trims spaces on this regex)

How expansive? The error if you just hit the blue button is pretty clear about the rules. If it messes with these rules then drop it and use the default?

Comment (chrisnojima) [reply shared/constants/signup.tsx]: just replace all the non a-Z-_

Comment (mlsteele) [reply shared/constants/signup.tsx]: My ipad is named `Miles's iPad` which is valid but that would turn it into `Miless iPad`

**Comment by mlsteele** [comment]
Drop personalized device name suggestions. And "My ". Pending an OK from design

**Comment by cjb** [comment]
@mlsteele you might want to back out @hyperobject's PR more, since it added a dep that's now unnecessary

**Comment by mlsteele** [comment]
Good point. How about doing that as a separate ticket? In order to not do it this week. And in case the decision is reversed.

**Comment by hyperobject** [comment]
@cjb @mlsteele don't think my pr actually added a dependency, we already included expo-constants previously

**Comment by mlsteele** [comment]
ready anew

**Pull request merged**